src/utils/sketches/fish.js

```diff
@@ -95,6 +97,22 @@ module.exports = p5 => {
     draw();
   };
 
+  p5.getKnnData = () => {
```

Comment:
nit: can we move this up in this file? This actually contains a lot of really important information that we might want to reference in the future and it's buried kind of deeply in here.

Reply:
yeah, good call -- i'll move it up to the top of our p5 sketch function